src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -12,6 +12,8 @@
 
 namespace Stryker.CLI.UnitTest
 {
+    using System.Linq;
```

Comment:
We place usings outside of the namespace

Reply:
Ahhhh crap, sorry it's my resharper file that is bugging! Let me change it quickly ;-)

Reply:
No worries :)

Reply:
Fixed!